Options.py

```diff
@@ -102,7 +112,7 @@ def meta__init__(self, *args, **kwargs):
 class Option(typing.Generic[T], metaclass=AssembleOptions):
     value: T
     default: typing.ClassVar[typing.Any]  # something that __init__ will be able to convert to the correct type
-
+    visibility = Visibility.all
```

Comment:
```suggestion
    visibility = Visibility.all

```
I think the empty line that was here was valuable for the contextualizing of the comment below it.
Without the empty line, it's a little more difficult to understand what the comment is about.